airflow/jobs.py

```diff
@@ -1434,7 +1434,7 @@ def _execute_helper(self, processor_manager):
                                                           State.NONE)
 
                 self._execute_task_instances(simple_dag_bag,
-                                             (State.SCHEDULED,))
```

Comment:
an alternative approach I came up with had it set the state to `State.NONE` at https://github.com/apache/incubator-airflow/blob/master/airflow/models.py#L1291 if it was already `State.QUEUED` (which makes the scheduler pick it back up to work on the next time around). but that seemed hackish, as you'd see the task moved from queued to none to queued to none before it finally would get worked on.